### DISCUSSION TITLE
Ask : weather manager

### DISCUSSION THREAD

**Discussion by Ananas-Kuchen** [kylecorry31/Trail-Sense#713] [Q&A]
I've read the user guide in the application. but when I checked in the settings there was no barometer. I just found the altimeter and thermometer from the instructions written

I want to activate "sea level pressure" in order to use the weather manager

Does it not all phones have barometer support or what?

I saw many times in the screenshot (fdroid) but could not find the tab showing the weather icon when I installed it

Thanks

**Comment by kylecorry31** [comment]
Most phone's do not have the barometer which is a requirement for the weather tool. The altimeter can use GPS as a fallback. If you do not see the weather tab, you do not have a barometer.

I'll give the app another pass through to make sure I hide all weather references on devices without barometers - it's hard for me to test that because every device I use has a barometer.

**Reply by Ananas-Kuchen** [reply]
I know it. apparently because my phone doesn't support barometer lol

This is strange, I did not find the latest phone info that has barometer support. until I found this https://m.gsmarena.com/samsung_galaxy_nexus_i9250-4219.php

Anyway thank you for the answer with kindness, sir

**Reply by Ananas-Kuchen** [reply]
Wait. So that's why the Weather Manager toggle can't be activated in the notification bar. that explains why

**Reply by kylecorry31** [reply]
Yes, that's exactly why (and I have this issue to hide the weather monitor toggle for the next release: #688)

**Reply by Lee-Carre** [reply]
> I'll give the app another pass through to make sure I hide all weather references on devices without barometers - it's hard for me to test that because every device I use has a barometer.

Mine lacks a barometer (much to my annoyance), so I'll look out for orphan references (other than describing the feature for devices with a barometer) during usage / tinkering, and report any I find.
